topics/assembly/tutorials/get-started-genome-assembly/slides.html

```diff
@@ -0,0 +1,395 @@
+]
+
+**Higher heterozygosity -> harder to assemble => Increase of sequencing depth**
+
+.footnote[https://www.genome.gov/genetics-glossary/heterozygous]
+
+---
+
+### Genome information: Heterozygosity level
+
+.image-125[
+  ![Concepts in phased assemblies](../../images/phased-assemblies.png)
+]
+
+.footnote[Heng Li’s blog: lh3.github.io/2021/04/17/concepts-in-phased-assemblies]
+
+---
+
+### Genome information: Complexity aka repeats elements
+
+.left[
+**It is impossible to resolve repeats of length L unless you have reads longer than L**
+]
+
+Most common source of assembly errors: 
+ 
+.pull-left[
+.image-65[
+  ![Collapsed consensus from repeat copies](../../images/collapsed-consensus.png)
+]]
+
+.pull-right[
+.image-65[
+  ![Collapsed, excision and rearrangement consensus](../../images/excision-rearrangement-consensus.png)
+]]
+
+---
+
+### Genome information: Others
+
+- **Karyotype:** chromosome number
+- **Sex chromosome system:** None, XY, ZW, UV,…
+- **Purity:** possible presence of contaminants and/or symbionts?
+- Does any other **useful data (NCBI, SRA, ENA, etc)** that could improve my assembly exist?
+
+---
+
+### Genome information: Tips
+
+.pull-left[
+- **Flux cytometry** :  genome size and ploidy level
+- **k-mer frequency from Illumina reads** : genome size, ploidy level, GC content, heterozygosity and even repeats composition!
+]
+
+.pull-right[
+.image-20[
+  ![GenomeScope Profile](../../images/kmers-frequency.png)
+]]
+
+---
+
+### The best possible DNA
+
+.left[
+Select the best possible DNA source and extraction.
+The extraction of **high-quality DNA** is the **most important** aspect of a successful genome project
+
+The **lack of a good starting material** will limit the choice of sequencing technology and will **affect the quality of obtained data**
+]
+
+---
+
+### The best possible DNA: Chemical purity of DNA
+
+.left[
+**Sample-related contaminants:**
+- Polysaccharides
+- Proteoglycans
+- Proteins
+- Secondary metabolites
+- Polyphenols
+- Humic acids
+- Pigments
+- Etc,...
+
+All these contaminants **can impair the efficacy of library preparation** in any technology, especially true **for PCR-free libraries** (both PacBio and ONT)
+]
+
+---
+
+### The best possible DNA: Quantity of DNA
+
+.left[
+Different technologies require different amount of DNA:
+- Illumina and 10x > 3 ng
+- BioNano > 200 ng
+- ONT > 1 μg
+- Dovetail (Hic) > 5 μg
+- PacBio > 15 μg
+]
+
+---
+
+### The best possible DNA: Structural integrity of DNA
+
+.left[
+High Molecular Weight (**HMW**) for Nanopore/PacBio (obtained mainly from fresh material)
+]
+
+---
+
+### The best possible DNA: Tips
+
+.left[
+- Many *DNA extraction protocol* available for a lot of species/taxa (VGP, Darwin Tree of Life, Nanopore, PacBio, etc)
+- **Keep DNA sample** from the same individual in case of library preparation/sequencing fail, need more coverage, new sequencing technology, etc
```

Comment:
```suggestion
- **Keep DNA samples** from the same individual in case of library preparation/sequencing fail, need more coverage, new sequencing technology, etc
```